client-extensions/liferay-object-collection-filter-extension/src/common/components/search/SearchResult.js

```diff
@@ -0,0 +1,33 @@
+import { Body, Cell, Head, Row, Table } from "@clayui/core";
+import { useEffect, useState } from "react";
+
+const SearchResult = ({columnHeaders, items}) => {
+
+	return (
+        <Table>
+            <Head items={columnHeaders}>
+                {(column) => (
+                    <Cell key={column.value}>{column.label}</Cell>
+                )}
+            </Head>
+                <Body items={items}>
+                    {(row) => (
+                        <Row items={columnHeaders}>
+                            {(column) => (
+                                <Cell key={column.value}>{getCellValue(row[column.value])}</Cell>
+                            )}
+                        </Row>
+                    )}
+            </Body>
+        </Table>
+      );
+};
+
+export default SearchResult;
+
+const getCellValue = (value) => {
```

Comment:
I think here we should use switch case instead 
`
const getCellValue = (value) => {
  switch (typeof value) {
    case "string":
    case "number":
      return value;
    case "object":
      return value?.name || "";
    case "boolean":
      return value ? "True" : "False";
    default:
      return "";
  }
};
`